src/game/client/input/keyboard.cpp

```diff
@@ -194,25 +204,25 @@ void Keyboard::Reset_Keys()
 /**
  * 0x0040A460
  */
-bool Keyboard::Is_Shift()
+bool Keyboard::Is_Shift() const
 {
-    return m_modifiers & MODIFIER_SHIFT || m_modifiers & MODIFIER_SHIFTEX;
+    return m_modifiers & MODIFIER_LSHIFT || m_modifiers & MODIFIER_RSHIFT || m_modifiers & MODIFIER_SHIFTEX;
```

Comment:
I always find it best to add brackets to these to prevent any issues with operator precedence. In this instance there is no issues though.